tests/integration/deploy/test_managed_stack_deploy.py

```diff
@@ -0,0 +1,177 @@
+import os
+from samcli.lib.bootstrap.companion_stack.data_types import CompanionStack
```

Comment:
nit: This should be in the 3rd block of imports (with the other samcli * imports).